src/components/AddItem.js

```diff
@@ -10,12 +10,13 @@ const initialFormState = {
   lastPurchased: null,
 };
 
-const AddItem = ({ token }) => {
+const AddItem = ({ token, props }) => {
```

Comment:
@veeteeran is `props` still being used here? 
(by the way, this object you are destructuring _is_ props, this is like saying `props.props`, just FYI)

Reply:
We forgot to delete props. Definitely not supposed to be in there. Thanks Chris!